src/index.metadata.ts

```diff
@@ -21,4 +20,8 @@ export const V2077_DIR = `V2077`;
 export const V2077_GENERATED_MOD_NAME_TAG = ` (${EXTENSION_NAME_INTERNAL})`;
 export const V2077_GENERATED_MOD_VERSION_PRERELEASE = EXTENSION_NAME_INTERNAL;
 
+export const VORTEX_STORE_PATHS = {
+  settings: [`settings`, `v2077`],
+};
```

Comment:
When changing this, there are other places that need to be changed to support the different path in, I think, actions, reducers and the view. I don't see them in this commit, did they get maybe get missed on accident?